model/converter/json/fixtures/ui_01.json

```diff
@@ -70,7 +71,8 @@
       ],
       "logs": [],
       "processID": "p1",
-      "warnings": null
+      "warnings": null,
+      "incomplete": false
```

Comment:
why do we need to add this? Isn't false going to be the default value anyway?

Reply:
Yes, but the fixtures are read as bytes and compared on byte level with the output of an encoded trace. They are not transformed to an actual trace. This is why I need to add it.